manifest.json

```diff
@@ -30,7 +30,19 @@
     "background": {
             "scripts": ["js/background.js"]
     },
-    "permissions": [
-        "contextMenus"
-    ]
+    "chrome_settings_overrides": {
+      "search_provider": {
+		 "homepage": "http://www.duckduckgo.com",
+         "encoding": "UTF-8",
+         "favicon_url": "https://duckduckgo.com/favicon.ico",
+         "is_default": true,
+         "keyword": "duckduckgo.com",
+         "name": "DDG Search",
+         "search_url": "https://duckduckgo.com/?q={searchTerms}"
+      },
+	  "startup_pages": ["https://duckduckgo.com"]
```

Comment:
@AdamSC1-ddg Do we actually want this? I am not sure if showing `duckduckgo.com` on each start of the browser (while it is also the homepage) is not a bit too much //cc @bsstoner